purchases/src/main/kotlin/com/revenuecat/purchases/Backend.kt

```diff
@@ -142,13 +154,17 @@ internal class Backend(
                     postReceiptCallbacks.remove(cacheKey)
                 }?.forEach { (onSuccess, onError) ->
                     try {
+                        val attributeErrors =
+                            result.body?.takeIf { it.has(ATTRIBUTES_ERROR_RESPONSE_KEY) }
+                                ?.getJSONObject(ATTRIBUTES_ERROR_RESPONSE_KEY)?.getAttributeErrors()
+                                ?: emptyList()
                         if (result.isSuccessful()) {
-                            onSuccess(result.body!!.buildPurchaserInfo())
+                            onSuccess(result.body!!.buildPurchaserInfo(), attributeErrors)
                         } else {
-                            onError(result.toPurchasesError(), result.responseCode < HTTP_SERVER_ERROR_CODE)
+                            onError(result.toPurchasesError(), result.responseCode < HTTP_SERVER_ERROR_CODE, attributeErrors)
                         }
                     } catch (e: JSONException) {
-                        onError(e.toPurchasesError(), false)
+                        onError(e.toPurchasesError(), false, emptyList())
```

Comment:
same comment here regarding named params

Reply:
I can't put named params here because it's a typed function